estatioapp/app/src/main/java/org/estatio/module/assetfinancial/contributions/FixedAsset_newAccount.java

```diff
@@ -51,7 +51,7 @@ public FixedAssetFinancialAccount act(
         return fixedAssetFinancialAccountContributions.newAccount(this.fixedAsset, financialAccount);
```

Comment:
reviewed commit, looks ok.  The framework should fail to start if we miss out any of these accidentally